### ISSUE TITLE
Methods to control uploads

### ISSUE THREAD

**Issue by phoenixz** [Valums-File-Uploader/file-uploader#14]
I was using valums original file uploader, just saw it is no longer maintained and forked here..

BIG question here: How can I cancel uploads? It says in the description they are cancelable.. great.. how? :)

Thing is, I have a chat project where I use a very simple upload file directly to chat functionality. I have a drop down menu where this option (upload file) is available, and a click there gives the file upload requester.. Select a file, and a separate console comes up, where the file upload progress is shown. Once the file is finished, the console disappears again and the file is posted in the chat.

Problem is, if I want to cancel the upload, I'd need a cancel method somewhere. I have not found any. Not in the documentation at least, and currently I am using the original version, and what I have understood from the code is that it basically can only be canceled by clicking on an HTML element that has qq-uploader-cancel.. This is FAR from ideal obviously :) 

So in short, I'd like to see some method to control uploads.. It would be VERY nice to just have a JS library that I can control from, say, jQuery
